app/packages/addons-service/provider.ts

```diff
@@ -83,8 +87,8 @@ export default class Provider {
         return Boolean(this.configuredAddon);
     }
 
-    constructor(provider: any, currentUser: CurrentUserService, node: NodeModel) {
-        setOwner(this, getOwner(node));
+    constructor(provider: any, currentUser: CurrentUserService, node?: NodeModel) {
+        setOwner(this, getOwner(provider));
```

Comment:
Did typescript help you catch this, or did you get an error of some sort and had to figure out what it was from?

Reply:
I got an error when I tried to do `getOwner` on the node, now that it's an optional parameter (not used on user addons page)